pkg/rtc/rtc_test.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 the u-root Authors. All rights reserved
```

Comment:
All the tests in here are Linux specific. ioctl are specific to LInux and the QEMU VM tests are Linux-only at the moment. You can put everything here into rtc_linux_test.go.